site/content/shared/docs/latest/development_guidelines.md

```diff
@@ -88,6 +96,7 @@ Developers should feel free to add more structure as complexity grows by making
     * Generators can be run via hack/gen.sh
     * Kapp-controller’s aggregated API server has a separate generator: hack/gen-apiserver.sh
     * The CRD yaml is generated via[ a separate script](https://github.com/vmware-tanzu/carvel-kapp-controller/blob/85e814cda7109169809ede1c8a4f211739ad15d2/hack/gen-crds.sh) that is run by hack/build.sh
+* Imports: use the alias `metav1` when importing `"k8s.io/apimachinery/pkg/apis/meta/v1"`
```

Comment:
Does this apply to all imports or only to imports that convey versions like `v1` or `v1alpha1`?

Reply:
i think it applies to exactly this (very common in controllers) import.  I didn't mean to imply any wildcards in this regex, although i guess there's a similar rule for `.../core/v1` .     What should i change to make this more clear?

Reply:
My suggestion would be:

Imports: when the imported package is named after a version, the previous package should be added for more context or something to identify the intent (ex: ` metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"` or `regv1 "github.com/google/go-containerregistry/pkg/v1"`)